.github/workflows/python-package.yml

```diff
@@ -88,13 +88,15 @@ jobs:
           - '3.8'
           - '3.9'
           - '3.10'
+          - '3.11'
+          - '3.12-dev'
```

Comment:
I wonder if using a development version of Python will lead to frequent test failures.

Reply:
Failures are possible but hopefully not frequent. I've rarely seen them when testing dev versions, but I have found a release blocker in CPython itself by doing so :)

This may also help find early issues in Black and isort, and other linters.

It also means Darker can be ready for 3.12 sooner.

If there are failures, and they're not quick to fix, I wouldn't hesitate to comment this out until it's resolved.

I guess as the linters used by Darker do AST things, there's more exposure to syntax changes before the [beta feature freeze in May](https://peps.python.org/pep-0693/#schedule), so one approach could be to wait until then.